environment.yml

```diff
@@ -13,3 +13,16 @@ dependencies:
   - cartopy
   - arm_pyart
   - xarray
+  - pip
+  - pip:
+    - wheel
+    - watchdog
+    - flake8
```

Comment:
Do we want all of these in our user environment file?

Reply:
Yeah, if people are going to be developing with this, I think they should be? They are relatively straight forward packages to install (and well supported)